doc/nrf/releases/release-notes-latest.rst

```diff
@@ -144,6 +149,11 @@ sdk-nrfxlib
 
 See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for the most current information.
 
+Multiprotocol Service Layer (MPSL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Modified the Bluetooth controller for nRF5340 to use SWI by default, instead of EGU.
```

Comment:
This is about https://github.com/zephyrproject-rtos/zephyr/pull/27801, correct?
Why is this related to MPSL? To me it looks like an update to the Zephyr Bluetooth Controller.
So it should be moved down to the Zephyr > Drivers > Bluetooth Controller section.
Text suggestion:
Updated the Bluetooth Controller to use SWI instead of EGU for nRF5340.

Reply:
Leaving this to Bluetooth people.